example/src/main/resources/application.properties.example

```diff
@@ -66,3 +66,9 @@ springdoc.swagger-ui.operationsSorter=alpha
 springdoc.swagger-ui.tagsSorter=alpha
 springdoc.default-produces-media-type=application/json
 springdoc.swagger-ui.displayOperationId=true
+
+minio.endpoint=http://localhost:9000
```

Comment:
change minio to s3, will need to add to application-test.properties too
should also add to the docker-compose file minio service